tutorials/tutorial2/tutorial2.rb

```diff
@@ -0,0 +1,9 @@
+require 'date'
+
+def age(dob)
+  now = Time.now.utc.to_date
+  age =  now.year - dob.year - ((now.month > dob.month || (now.month == dob.month && now.day >= dob.day)) ? 0 : 1)
```

Comment:
- u need to check the user input , when user enter '2024' , ur system is display "minus age"